src/contour/helper.cpp

```diff
@@ -340,6 +340,15 @@ terminal::renderer::PageMargin computeMargin(terminal::ImageSize _cellSize,
     return {leftMargin, bottomMargin};
 }
 
+terminal::renderer::FontDescriptions sanitizeFontDescription(terminal::renderer::FontDescriptions _fonts, crispy::Point screenDPI)
```

Comment:
`crispy::Point screenDPI` -> `crispy::Point _screenDPI`

Reply:
This f**ks me my really hard. 😅  Because in Windows Terminal the naming convention is exactly the opposite. Every member of a class is `_member` and every paramter is `parameter`.

Reply:
haha. oh sorry. but to be honest, I am already in progress of changing that (in my head). Let's talk about it in Discord :)